### ISSUE TITLE
User is able to create project - but pipelines are not displayed in OSIO UI

### ISSUE THREAD

**Issue by ldimaggi** [openshiftio/openshift.io#383]
Started seeing this problem on July 20 - specific user account ("osiotest3142") is seeing this behavior:

* After deleting existing spaces, github repos, etc.
* Creating a new project results in the project deployed to stage - but no pipeline created.

![1](https://user-images.githubusercontent.com/642621/28447812-474fcdbc-6da1-11e7-914c-6d262d787f91.png)

![2](https://user-images.githubusercontent.com/642621/28447815-4760ebce-6da1-11e7-8167-bdcc0de36726.png)

![3](https://user-images.githubusercontent.com/642621/28447813-475f530e-6da1-11e7-9a92-0f8a9bbcf507.png)

![screenshot from 2017-07-20 23-06-59](https://user-images.githubusercontent.com/642621/28447808-472f2e90-6da1-11e7-8898-db5615519aa2.png)

![screenshot from 2017-07-20 23-00-50](https://user-images.githubusercontent.com/642621/28447811-4744ea8c-6da1-11e7-840c-3cb96a0cf367.png)

![screenshot from 2017-07-20 22-59-00](https://user-images.githubusercontent.com/642621/28447816-4761a582-6da1-11e7-85d3-8a46ef67aa49.png)

![screenshot from 2017-07-20 22-58-21](https://user-images.githubusercontent.com/642621/28447814-47609bb0-6da1-11e7-9cfe-f726718e405c.png)

![screenshot from 2017-07-20 23-24-44](https://user-images.githubusercontent.com/642621/28448001-a98d3ca2-6da2-11e7-9f98-fb3f328d6c39.png)

**Comment by jstrachan** [comment]
incidentally the pipeline is visible in the openshift console in the osiotest3142 namespace
https://console.starter-us-east-2.openshift.com/console/project/osiotest3142/browse/pipelines

it looks like the issue is that the space label wasn't associated with the Pipeline so its invisible to the fabric8 ui.

I just modified the BuildConfig directly to add `space:  testjuly20` as a label and it now appears.

So looks like there's an issue with the Forge wizard client not populating the labelSpace with `testjuly20` so that it gets added to the BuildConfig labels

**Comment by jstrachan** [comment]
this is the underlying cause and required fix https://github.com/fabric8-ui/fabric8-ui/issues/1770

**Comment by ldimaggi** [comment]
Thanks James - I looked in the other projects, but missed that!

Question - Is there a separate issue here - that pipelines are NOT being created for the run project?

**Comment by maxandersen** [comment]
looks like a regresssion and blocking new users. putting on killlist.

**Comment by jstrachan** [comment]
the workaround for now from the CLI is
```
oc project ${TENANT}
oc label bc pipelineName space=mySpaceName
```

**Comment by jyasveer** [comment]
This is also blocking us to see the stack reports from the pipeline page.

**Comment by ldimaggi** [comment]
The workaround is not working for me:

![screenshot from 2017-07-24 19-24-28](https://user-images.githubusercontent.com/642621/28549081-f365025a-70a5-11e7-9f5a-3d84f9f26281.png)
![screenshot from 2017-07-24 19-24-14](https://user-images.githubusercontent.com/642621/28549082-f37fe1e2-70a5-11e7-941d-17768b1eeff1.png)
![screenshot from 2017-07-24 19-23-50](https://user-images.githubusercontent.com/642621/28549086-f385a3d4-70a5-11e7-89ca-baa6e485a2c7.png)
![screenshot from 2017-07-24 19-23-26](https://user-images.githubusercontent.com/642621/28549084-f38035b6-70a5-11e7-8993-b8634cd77a5a.png)
![screenshot from 2017-07-24 19-22-44](https://user-images.githubusercontent.com/642621/28549083-f37ff646-70a5-11e7-95bf-ba6f6ce91039.png)
![screenshot from 2017-07-24 19-22-21](https://user-images.githubusercontent.com/642621/28549085-f3806e64-70a5-11e7-93be-01c4cd592e5b.png)

**Comment by joshuawilson** [comment]
@jstrachan will you be able to fix this? Do you know what is causing it?

**Comment by ldimaggi** [comment]
This problem seems to be reproducible - in my account - pipelines are being created - but builds are not being performed.

**Comment by jstrachan** [comment]
but you started this issue demonstrating that pipeilnes were working and the apps provisioned? Are you now saying this has mutated into a totally separate issue now?

**Comment by jstrachan** [comment]
@ldimaggi you say the workaround doesn't work for you - but then show that pipelines are visible after you label them? am kinda confused now

@joshuawilson apart from recent confusion with @ldimaggi 's recent comments, I'm assuming this whole issue is  fabric8-ui/fabric8-ui#1770 i.e. a UI regression and nothing to do with jenkins / forge

**Comment by ldimaggi** [comment]
Sorry about the confusion - the workaround does cause the pipeline to appear for me.

Actually using the pipeline is blocked for me by https://github.com/openshiftio/openshift.io/issues/410

**Comment by joshuawilson** [comment]
@jstrachan I was thinking it was  fabric8-ui/fabric8-ui#1770 too.

**Comment by ldimaggi** [comment]
The problem was caused by out of date tenant service. Closing.

See:  https://github.com/openshiftio/openshift.io/issues/411

**Comment by ldimaggi** [comment]
Sorry! Closed by mistake!

**Comment by jstrachan** [comment]
OK I think this is now fixed in prod - can folks try and reproduce again (with a recent tenant).

The trick I found to reproduce it was from the home page created a new space, then immediately add an app to it from the same wizard - previously forge was losing the space label but now appears to work correctly again. The fix was to avoid a combo box / cache and to avoid rejected valid space names that are newly created. I was able to reproduce - but now in prod I can't
